types/ascoel/PB868LR/uplink.js

```diff
@@ -0,0 +1,73 @@
+function hex2ascii(hexx) {
+  const hex = hexx.toString(); // force conversion
+  let str = "";
+  for (let i = 0; i < hex.length; i += 2) {
+    str += String.fromCharCode(parseInt(hex.substr(i, 2), 16));
+  }
+  return str;
+}
+
+function consume(event) {
+  const payload = event.data.payloadHex;
+  const { port } = event.data;
+  const bits = Bits.hexToBits(payload);
+  const data = {};
+  let topic = "default";
+
+  switch (port) {
+    case 2:
+      data.downlinkNr = Bits.bitsToUnsigned(bits.substr(0, 32));
+      topic = "debug";
+      break;
+    case 5:
+      if (payload === "50423836384C5248") {
```

Comment:
is this really how to recognise sensor model?

Reply:
<img width="601" alt="image" src="https://user-images.githubusercontent.com/73245684/204532779-865c7c69-c295-4fce-9376-ccb72371157e.png">